scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -31,55 +31,56 @@
 	topicClassVocab	Controlled Vocabulary Name	The controlled vocabulary used for the keyword term (e.g. LCSH, MeSH)		text	27	(#VALUE)	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	topicClassification	citation	
 	topicClassVocabURI	Controlled Vocabulary URL	The URL where one can access information about the term's controlled vocabulary	https://	url	28	<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	topicClassification	citation	
 	publication	Related Publication	The article or report that uses the data in the Dataset. The full list of related publications will be displayed on the metadata tab		none	29		FALSE	FALSE	TRUE	FALSE	TRUE	FALSE		citation	http://purl.org/dc/terms/isReferencedBy
-	publicationCitation	Citation	The full bibliographic citation for the related publication		textbox	30	#VALUE	TRUE	FALSE	FALSE	FALSE	TRUE	FALSE	publication	citation	http://purl.org/dc/terms/bibliographicCitation
-	publicationIDType	Identifier Type	The type of identifier that uniquely identifies a related publication		text	31	#VALUE: 	TRUE	TRUE	FALSE	FALSE	TRUE	FALSE	publication	citation	http://purl.org/spar/datacite/ResourceIdentifierScheme
-	publicationIDNumber	Identifier	The identifier for a related publication		text	32	#VALUE	TRUE	FALSE	FALSE	FALSE	TRUE	FALSE	publication	citation	http://purl.org/spar/datacite/ResourceIdentifier
-	publicationURL	URL	The URL form of the identifier entered in the Identifier field, e.g. the DOI URL if a DOI was entered in the Identifier field. Used to display what was entered in the ID Type and ID Number fields as a link. If what was entered in the Identifier field has no URL form, the URL of the publication webpage is used, e.g. a journal article webpage	https://	url	33	<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>	FALSE	FALSE	FALSE	FALSE	TRUE	FALSE	publication	citation	https://schema.org/distribution
-	notesText	Notes	Additional information about the Dataset		textbox	34		FALSE	FALSE	FALSE	FALSE	TRUE	FALSE		citation	
-	language	Language	A language that the Dataset's files is written in		text	35		TRUE	TRUE	TRUE	TRUE	FALSE	FALSE		citation	http://purl.org/dc/terms/language
-	producer	Producer	The entity, such a person or organization, managing the finances or other administrative processes involved in the creation of the Dataset		none	36		FALSE	FALSE	TRUE	FALSE	FALSE	FALSE		citation	
```

Comment:
Why is this diff so big? Can you please summarize what changed?

Reply:
The one new field was added and the displayOrder for everything after that increases by 1

Reply:
Ok, so all the change above must be whitespace, I guess.

Reply:
not white space, but pretty trivial - the publicationCitation line, for example, now has a displayOrder of 31 instead of 30. Same all the way down - just a change to that one number in each line.